src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/SamlAuthenticationClaimResource.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Runtime.Serialization;
+using SamlAuthorityBinding = Microsoft.IdentityModel.Tokens.Saml.SamlAuthorityBinding;
+
+namespace CoreWCF.IdentityModel.Tokens
+{
+    [DataContract]
+    internal class SamlAuthenticationClaimResource
+    {
+        [DataMember]
+        private readonly DateTime _authenticationInstant;
+        [DataMember]
```

Comment:
Anything with a DataMember attribute means it's able to be serialized, in which case naming is important. We also ideally want to have it compatible with NetFx. Modify each of these DataMember attributes to have a Name parameter specifying the name originally used on NetFx (without the underscore). Also the DataContract attribute needs to be modified to specify the same Namespace as was used on NetFx. That would be `"[http://schemas.datacontract.org/2004/07/System.IdentityModel.Tokens](http://schemas.datacontract.org/2004/07/System.IdentityModel.Tokens)"`